include/dlaf/util_cublas.h

```diff
@@ -0,0 +1,110 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2020, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#ifdef DLAF_WITH_CUDA
+
+#include <cublas_v2.h>
+#include <blas.hh>
+
+// TODO: Which namespace?
+namespace dlaf {
+namespace internal {
+namespace gpu {
+
+// TODO: Are these necessary?
+inline constexpr cublasSideMode_t blas_to_cublas_side(blas::Side side) {
```

Comment:
Perhaps you can overload `blas_to_cublas()` and remove the suffixes `_side`, `_uplo`, etc. 

Also an alternative name for the function could be `blas2cublas()`, in case someone prefers it. I am OK with the current name too.

Reply:
I'm indifferent about the particular name. Is there precedent for `x2y`? The `_type` suffix is indeed unnecessary, thanks! I'll remove that.

Reply:
I've removed the suffixes for the enumeration conversion functions. In principle `blas_to_cublas_cast` could also do without the `_cast` suffix. I slightly prefer to have it there to explicitly say that we're casting and returning the same underlying value in a different form, rather than a new value, but I can just as well go without the suffix as well.

Reply:
> I've removed the suffixes for the enumeration conversion functions. In principle blas_to_cublas_cast could also do without the _cast suffix. I slightly prefer to have it there to explicitly say that we're casting and returning the same underlying value in a different form, rather than a new value, but I can just as well go without the suffix as well.

I also think it's worth keeping the suffix for that one.

> I'm indifferent about the particular name. Is there precedent for x2y

I think that's some UNIX-sm I may have subconsciously picked up (utilities like pdf2svg, etc). The current name is pretty clear, let's just keep it as it is.